recipe/meta.yaml

```diff
@@ -41,11 +42,13 @@ requirements:
     - pycurl
     - python-dateutil
     - requests
+    - packaging
     - sqlalchemy >=1.1
     - tornado >=5.1
     - traitlets >=4.3.2
     - jupyter_telemetry >=0.1.0
-
+  run_constrained:
```

Comment:
This could be removed as it is covered in the run section.

Reply:
Great, removed